files/default/allowed_functions.txt

```diff
@@ -0,0 +1,287 @@
+array_ndims
+array_position
+array_positions
+array_prepend
+array_remove
+array_replace
+array_to_json
+array_to_string
+array_to_tsvector
+array_upper
+avg
+bit_and
+bit_or
+bool_and
+bool_or
+bound_box
+box
+broadcast
+btrim
+cardinality
+cbrt
+ceil
+ceiling
+center
+chr
+circle
+clock_timestamp
+concat
+concat_ws
+convert
+convert_from
+convert_to
+corr
+cos
+cosd
+cot
+cotd
+count
+covar_pop
+covar_samp
+cume_dist
+date_part
+date_pl_interval
+date_pli
+date_recv
+date_send
+date_smaller
+date_sortsupport
+date_trunc
+decode
+degrees
+dense_rank
+diagonal
+diameter
+div
+encode
+enum_cmp
+enum_eq
+enum_first
+enum_ge
+enum_gt
+enum_in
+enum_larger
+enum_last
+enum_le
+enum_lt
+enum_ne
+enum_range
+enum_smaller
+exp
+factorial
+family
+first_value
+floor
+format
+generate_series
+get_bit
+get_byte
+get_current_ts_config
+height
+host
+hostmask
+inet_merge
+inet_same_family
+initcap
+isclosed
+isempty
+isfinite
+isopen
+json_agg
+json_array_element
+json_array_element_text
+json_array_elements
+json_array_elements_text
+json_array_length
+json_build_array
+json_build_object
+json_each
+json_each_text
+json_extract_path
+json_extract_path_text
+json_object_agg
+json_object_keys
+json_populate_record
+json_populate_recordset
+jsonb_agg
+jsonb_array_element
+jsonb_array_element_text
+jsonb_array_elements
+jsonb_array_elements_text
+jsonb_array_length
+jsonb_build_array
+jsonb_build_object
+jsonb_extract_path
+jsonb_extract_path_text
+jsonb_insert
+jsonb_pretty
+jsonb_set
+jsonb_strip_nulls
+jsonb_typeof
+justify_days
+justify_hours
+justify_interval
+lag
+last_value
+lead
+left
+length
+line
+ln
+log
+lower
+lower_inc
+lower_inf
+lpad
+lseg
+ltrim
+macaddr8_set7bit
+make_date
+make_interval
+make_time
+make_timestamp
+make_timestamptz
+masklen
+max
+md5
+min
+mod
+mode
+network
+now
+npoints
+nth_value
+ntile
+num_nonnulls
+num_nulls
+numnode
+parse_ident
+path
+pclose
+percent_rank
+percentile_cont
+percentile_disc
+phraseto_tsquery
+pi
+plainto_tsquery
+point
+polygon
+popen
+power
+querytree
+quote_ident
+quote_literal
+quote_nullable
+radians
+radius
+random
+range_merge
+rank
+regexp_match
+regexp_matches
+regexp_replace
+regexp_split_to_array
+regexp_split_to_table
+regr_avgx
+regr_avgy
+regr_count
+regr_intercept
+regr_r2
+regr_slope
+regr_sxx
+regr_sxy
+regr_syy
+repeat
+replace
+reverse
+right
+round
+row_number
+row_to_json
+rpad
+rtrim
+scale
+set_bit
+set_byte
+set_masklen
+setweight
+sign
+sin
+sind
+slope
+split_part
+sqrt
+statement_timestamp
+stddev
+stddev_pop
+stddev_samp
+string_agg
+string_to_array
+strip
+strpos
+substr
+substring
+sum
+tan
+tand
+text
+to_ascii
+to_char
+to_date
+to_hex
+to_json
+to_jsonb
+to_timestamp
+to_tsquery
+to_tsvector
+transaction_timestamp
+translate
+trunc
+ts_delete
+ts_filter
+ts_headline
+ts_rank
+ts_rewrite
+tsvector_to_array
+unnest
+upper
+upper_inc
+upper_inf
+var_pop
+var_samp
+variance
+width
+width_bucket
+xmlagg
+xmlcomment
+xmlconcat
+xpath
+xpath_exists
+COUNT
+ictdashboards_tables
```

Comment:
can you also add capitalized version of this. ICTDASHBOARDS_TABLES as the whitelist is case sensitive

Reply:
can it take comments # something and still work?

Reply:
I'm raising a pull request on CKAN core to make the whitelist case-insensitive. For now, I'm pretty sure the ICT dashboard only needs the lowercase version.

Yes, it looks like comments are allowed.